### PR TITLE
Docs: Improved and fixed statistics section

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.
Ready for review and merge.

### CODE REVIEW

File: pywbem/_statistics.py [hunk 1]
```diff
@@ -45,7 +46,95 @@
...
+  user is calling.
+
+* Server times: The elapsed times for processing the WBEM operations in the
+  WBEM server from receiving the CIM-XML request message to sending back the
+  CIM-XML response message. The server times are not measured by pywbem, but
+  are taken from the `WBEMServerResponseTime` HTTP header field of a CIM-XML
+  response, if present. See :term:`DSP0200` for a description of this header
+  field.
+
+  The `WBEMServerResponseTime` HTTP header field is optional to be implemented
```
Comment (KSchopmeyer) [inline pywbem/_statistics.py]: Change "optional to be implemented" to "optionally implemented"

File: pywbem/_statistics.py [hunk 2]
```diff
@@ -141,25 +221,26 @@ def container(self):
...
+        receiving the response).
         """
         return self._exception_count
 
     @property
     def avg_time(self):
         """
-        :class:`py:float`: The average elapsed time for invoking the measured
-        operations, in seconds.
+        :class:`py:float`: The average elapsed client time for invoking the
```
Comment (KSchopmeyer) [inline pywbem/_statistics.py]: I think it sounds better if we change the phrase "for invoking" to " for execution of" so it becomse "for execution of the measured operations"  This applies to the min_time and max time also

Comment (andy-maier) [reply pywbem/_statistics.py]: DONE

File: pywbem/_statistics.py (hunk 1 above)
Comment (andy-maier) [reply pywbem/_statistics.py]: DONE

**Pull request merged**